File: src/loop/fib_loop.jl
function fib(n)
    n1 = BigInt(1)
    n2 = BigInt(1)
    for i in 1:(n-2)
        n1, n2 = n2, n1 + n2
    end
    n2
end

n = parse(Int64, ARGS[1])
println("res = ", fib(n))

File: src/loop/fib_loop.py
import sys


def fib(n):
    n1 = 1
    n2 = 1
    for i in range(n-2):
        n1, n2 = n2, n1+n2
    return n2

if __name__ == '__main__':
    n = int(sys.argv[1])
    print("res = {}".format(fib(n)))
